examples/sdk-snapgrid/src/components/Navbar/Navbar.module.css

```
@@ -0,0 +1,250 @@
+  text-decoration: none;
+}
+
+.logoMark {
+  font-size: 20px;
+  color: var(--gold);
+  line-height: 1;
+  filter: drop-shadow(0 0 8px rgba(201, 168, 76, 0.6));
+  animation: pulseMark 3s ease-in-out infinite;
+}
+
+@keyframes pulseMark {
```

Stylelint (17.10.0)

[error] Expected keyframe name "pulseMark" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)